communities.json

```diff
@@ -771,6 +771,13 @@
       "quote": "Discord has been amazing for accelerating the development of React Spring and React Three Fiber. Screen sharing and instant communication have been pivotal in resolving bugs and coordinating with our contributors.",
       "inviteCode": "poimandres",
       "githubUrl": "https://github.com/pmndrs"
+    },
+    {
+      "logo": "urllib3.svg",
+      "title": "urllib3",
+      "quote": "The urllib3 team uses Discord to answer questions from users and to collaborate on development tasks.",
+      "inviteCode": "urllib3",
```

Comment:
It looks like this invite code doesn't currently exist. I cannot identify your server without a valid invite code.

Reply:
Oh sorry! I thought this was an aspirational field :)

Reply:
Updated!

Reply:
Haha sorry, I should have just said to comment it. I've turned on vanity urls so you could set it back now